packages/device-utils/src/modelUtils.ts

```diff
@@ -1,14 +1,28 @@
 import { PartialDevice } from './types';
 
 export enum DeviceModel {
```

Comment:
will we create device-types package or will Connect depend on @trezor/device-utils? 🤔